orderedmap_test.go

```diff
@@ -304,6 +307,92 @@ func TestOrderedMap_Back(t *testing.T) {
 	})
 }
 
+func TestOrderedMap_MarshalJSON(t *testing.T) {
+	t.Run("MarshalJsonIntKeyValue", func(t *testing.T) {
+		m := orderedmap.NewOrderedMap()
+		m.Set(1, 1)
+		b, err := json.Marshal(m)
+		result := bytes.Equal([]byte(`[1,1]`), b) && nil == err
```

Comment:
This is a very opinionated way to serialize that I don't think is suitable for general use. If I remember correctly, I did not include JSON support in orderedmap because serialization is ambiguous (since maps aren't always ordered).

If you need to serialize in this specific format you should create an alias type. I cannot accept this patch, sorry.